src/main/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImpl.java

```diff
@@ -258,6 +359,29 @@ protected String getCreateAccountingDocumentReportEmailAddress() {
                 CuFPParameterConstants.CreateAccountingDocumentService.CREATE_ACCT_DOC_REPORT_EMAIL_ADDRESS);
     }
     
+    protected String getDuplicateFileReportEmailAddress() {

```

Comment:
I think we want DUPLICATE_FILE_REPORT_EMAIL_ADDRESS to have a list of addresses.  We want the developers to know because there may be something technically going awry, and the functionals (FIS / Cathy) would also want to know